src/popup/components/form/FormNumberSelect.vue

```diff
@@ -4,6 +4,7 @@
       v-if="modelValue"
       :value="modelValue"
       class="number-select-input"
+      aria-label="choose number of signers"
```

Comment:
This element is not specific to selecting number of signers and this would look weird in another context